### DISCUSSION TITLE
Need some help with hexo injector

### DISCUSSION THREAD

**Discussion by shmilyty** [hexojs/hexo#4941] [Question / Answer] [answered]
I tried to add an [alplayer](https://aplayer.js.org/) into my hexo blog with hexo injector, but there seems to have something wrong. The injector doesn't work at all.

Here're my config files.
![image](https://user-images.githubusercontent.com/61276699/163170471-81d5574b-2ac8-4def-a8f3-43590bdda8d8.png)
![image](https://user-images.githubusercontent.com/61276699/163170486-df27849e-1e3c-4ce9-bc3b-7c436d39f1aa.png)
There should be a music bar at the bottom of the page if I run hexo, there's nothing instead.

The localhost after `hexo clean+g+s`.
![image](https://user-images.githubusercontent.com/61276699/163170766-313c5a4b-511e-48b6-91ee-3acac2d3cf36.png)
I looked through the console. Fortunately, I found some error.
![image](https://user-images.githubusercontent.com/61276699/163170889-fdd80f3d-ca07-4ce9-bb94-423c03331e17.png)

**The sticky problem is I don't know how to deal with the error.** I tried to ask my friends and other people who's good at this field, but I haven't had any satisfied answers yet. That's why I start a new discussion here.

Thank you for watching. Plz, go ahead to save me, if you'd like to spend a little time on it.

---
These are some sources with the problem.
- [tutorial to the injector, which I copy the code and follow the instructions exactly the same](https://hexo.fluid-dev.com/posts/hexo-injector/#aplayer-%E9%9F%B3%E4%B9%90%E6%92%AD%E6%94%BE%E5%99%A8)
- [I opened an issue on the theme I'm current using, but the developer's suggestion can't help me.](https://github.com/fluid-dev/hexo-theme-fluid/issues/700)

**Comment by stevenjoezhang** [comment, marked as answer]
Try the following settings in Hexo `_config.yml`

```yml
skip_render:
  - js/aplayer.js
```

See also https://github.com/hexojs/hexo/issues/4468

**Reply by shmilyty** [reply]
It works!

Thank you very much.
